### ISSUE TITLE
[feature request] sync saves over network

### ISSUE THREAD

**Issue by papa-gene** [BernardoGiordano/Checkpoint#64]
Some people have more than one 3DS device, and like switching between them. It would be interesting to look into a way of syncing saves for a game over the network somehow.

A few ways to achieve this could be by means of a 3rd party storage system like GDrive or Dropbox, or by having the app sync between both systems directly.

Thank you for considering this request!

**Comment by Hugobert** [comment]
Just an idea: On the Nintendo Switch, this might potentially be achieved with a sysmodule that runs in the background or even in sleep mode.

**Comment by inarius** [comment]
Yes! I'd love to see this for Nintendo Switch, even if it needed to be initiated at a button press within homebrew. I know the files are slow to seek through and copy, so maybe a simple index like a SQLLite database could be used to keep track of what is in sync and make the operation faster.

**Comment by fennectech** [comment]
I’d love to see Dropbox support.

**Comment by tbsp** [comment]
It'd be great if the save/extra data for specific games could be marked for cloud sync so you could fire up Checkpoint and hit a few buttons to quickly sync your commonly synced games.
